nifi-extension-bundles/nifi-websocket-bundle/nifi-websocket-services-jetty/src/main/java/org/apache/nifi/websocket/jetty/RoutingWebSocketListener.java

```diff
@@ -27,6 +27,7 @@
 public class RoutingWebSocketListener extends AbstractAutoDemanding {
     private final WebSocketMessageRouter router;
     private String sessionId;
+    private Boolean secure;
```

Comment:
Is this nullable? The set method only takes a primitive `boolean`.

Reply:
Yeah, a left over of different approaches I tried. Fixed. Thanks!